### DISCUSSION TITLE
revalidate specific props

### DISCUSSION THREAD

**Discussion by raczosala** [vercel/next.js#16972] [Help]
Let's say we are fetching two different sets of data in our static props function, is there a way to use incremental static regeneration but specify a different revalidation for each data set?

```
export function getStaticProps(context) {
   const data = fetch(); // this data could be large and changes every week
   const data1 = fetch(); // this data is small and changes every couple of minutes or so

  return {
    data, // can we revalidate every 691200 seconds?
    data1, // can we revalidate every 60 seconds?
    revalidate: 60,
  }
}
```

currently both data sets will be fetched?

I guess the large data set could be cached and make it unexpressive to retrieve but still

**Comment by cooperwfloyd** [comment]
+1 would appreciate this feature as well. Running into use cases where I want some static props to use ISR but to change less often or not change at all after build.
